pytest_container/pod.py

```diff
@@ -143,7 +143,8 @@ def launch_pod(self) -> None:
         """
         runtime = get_selected_runtime()
         create_cmd = [runtime.runner_binary, "pod", "create"] + (
-            ["--name", self.pod_name] if self.pod_name else []
+            ["--name", self.pod_name] if self.pod_name else [] +
+            self.extra_pod_create_args
         )
```

Comment:
I think mypy is complaining about this
```suggestion
            ["--name", self.pod_name] if self.pod_name else []) + self.extra_pod_create_args
```